_episodes/07-errors.md

```diff
@@ -29,7 +29,26 @@ they become much easier to fix.
 
 Errors in Python have a very specific form,
 called a [traceback]({{ page.root }}/reference/#traceback).
-Let's examine one:
+A traceback can contain a lot of information, so the following
+schematic highlights the main sections of a traceback.
```

Comment:
I like the idea of a diagram to explain tracebacks, but I have to admit that I found this diagram more confusing than enlightening: I didn't understand what the red arrows were pointing from or to.